docs/config.rst

```diff
@@ -110,6 +114,21 @@ Java Kafka clients.
   * Default: 50
   * Importance: low
 
+
+``fetch.max.wait.ms``
+  Maximum amount of time the proxy will wait before returning a consumer response. Note that a response will be returned earlier if ``fetch.min.bytes`` is fulfilled.
+
+  * Type: int
+  * Default: 1000
+  * Importance: medium
+
+``fetch.min.bytes``
+  Minimum number of bytes in message keys and values returned by a single request before the timeout of `fetch.max.wait.ms` passes.
+
+  * Type: int
+  * Default: 1000
```

Comment:
That's a good point. I frankly don't remember why I chose 1000.

I agree that 1 is the ideal default value.

My concern now is that we might be changing the behavior of the proxy. Since we plan on releasing this in a minor version `4.1.x`, this might not be a good thing to do. 
Users up until now most likely tuned `consumer.request.max.bytes` to receive a certain payload from the proxy every time. If we were to change this behavior to return at the first sight of data, users' applications might start issuing requests more frequently than hoped for which could bring up issues like higher CPU use.

Even though it is a bit of an indirect change, I'm wondering if it is worth it to disable the behavior by default until 6.0